### PR TITLE
Make symbol listing work when cross-compiling

### PR DESCRIPTION
**Pull request by tamird**
@jasone see https://github.com/jemalloc/jemalloc/pull/818#issuecomment-313485737

### CODE REVIEW

**Comment by davidtgoldblatt** [comment]
Thanks for the diff!

File: configure.ac [hunk 1]
```diff
@@ -8,7 +8,7 @@ dnl ============================================================================
 dnl Custom macro definitions.
 
 dnl JE_CONCAT_VVV(r, a, b)
-dnl
```
Comment (interwq) [inline configure.ac]: @jasone: I think you mentioned the space after dnl was required, but couldn't find exactly why. Is this going to be an issue?

Comment (jasone) [inline configure.ac]: ```dnl ``` is an m4 comment delimiter.  ```dnla``` is not a comment delimiter, and IIRC best practice is to always follow ```dnl``` by a space.

**Comment by interwq** [comment]
@davidtgoldblatt: I'll level the dnl space part to you then (link #950 as a reminder).

**Pull request merged**